manifests/image-references

```diff
@@ -0,0 +1,12 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: openshift-apiserver-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-cluster-openshift-apiserver-operator:latest
+  - name: openshift-apiserver
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin:v3.10.0
```

Comment:
Your string values need to match - this is a find and replace, so it's erroring out because you don't declare "docker.io/openshift/origin:v3.10.0" anywhere.

Also, why aren't you using 4.0?